packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/petrinet/Species.java

```diff
@@ -0,0 +1,24 @@
+package software.uncharted.terarium.hmiserver.models.petrinet;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import lombok.Data;
+
+import java.io.Serializable;
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonDeserialize(using = SpeciesJsonDeserializer.class)
+public class Species implements Serializable {
+
+	@JsonAlias("sname")
+	private String name;
```

Comment:
Per discussion, it is hard to reason with petrinet acsets, and it would be good if we keep the naming as close as possible to the transport format. That is calling it sname instead of name.